### PR TITLE
Upgrade antd to v4

### PR DESCRIPTION
**Pull request by KaiVolland**
## Description

This upgrades `antd` to version `4`. It also introduces
- `@ant-design/icons` (replaces `Icon` from antd)
-  `@babel/runtime` (its polyfills are need for the tests)

Related changes to components and tests are needed to fit the new `antd` API.

## Pull request type

Please check the type of change your PR introduces:

- [x] Dependency updates

## Do you introduce a breaking change?

- [x] Yes

### CODE REVIEW

File: src/Component/DataInput/DataLoader/DataLoader.tsx [hunk 1]
```diff
@@ -122,15 +124,15 @@ export class DataLoader extends React.Component<DataLoaderProps, DataLoaderState
       activeParser.readData(JSON.parse(fileContent))
         .then((data: VectorData) => {
           this.props.onDataRead(data);
-          uploadObject.onSuccess(null, uploadObject.file);
+          // uploadObject.onSuccess(null, uploadObject.file);
```
Comment (jansule) [inline src/Component/DataInput/DataLoader/DataLoader.tsx:127]: please remove if not needed

File: src/Component/DataInput/DataLoader/DataLoader.tsx [hunk 2]
```diff
@@ -144,7 +146,7 @@ export class DataLoader extends React.Component<DataLoaderProps, DataLoaderState
       activeParser.readData(reader.result)
       .then((data: VectorData) => {
           this.props.onDataRead(data);
-          uploadObject.onSuccess(null, uploadObject.file);
+          // uploadObject.onSuccess(null, uploadObject.file);
```
Comment (jansule) [inline src/Component/DataInput/DataLoader/DataLoader.tsx:149]: please remove if not needed

File: src/Component/DataInput/StyleLoader/StyleLoader.tsx [hunk 3]
```diff
@@ -106,7 +107,7 @@ export class StyleLoader extends React.Component<StyleLoaderProps, StyleLoaderSt
 
     try {
       const style = await activeParser.readStyle(fileContent);
-      uploadObject.onSuccess(uploadObject.file);
+      // uploadObject.onSuccess(uploadObject.file);
```
Comment (jansule) [inline src/Component/DataInput/StyleLoader/StyleLoader.tsx:110]: please remove if not needed

File: src/Component/Symbolizer/Field/FontPicker/FontPicker.tsx [hunk 4]
```diff
@@ -67,7 +67,7 @@ export class FontPicker extends React.Component<FontPickerProps> {
     const children: JSX.Element[] = [];
     if (fontOptions) {
       fontOptions.forEach(fontOpt => {
-        children.push(<Option key={fontOpt}>{fontOpt}</Option>);
+        children.push(<Option value={fontOpt} key={fontOpt}>{fontOpt}</Option>);
```
Comment (jansule) [inline src/Component/Symbolizer/Field/FontPicker/FontPicker.tsx:70]: why don't we pass the options via `options` prop (like in `ChannelSelectionField`)?

File: src/Component/Symbolizer/Field/LineDashField/LineDashField.spec.tsx [hunk 5]
```diff
@@ -50,9 +50,9 @@ describe('OffsetField', () => {
   it('renders correctly', () => {
     expect(wrapper).not.toBeUndefined();
     const buttons = wrapper.find('Button');
-    const numberInputs = wrapper.find('InputNumber');
+    // const numberInputs = wrapper.find('InputNumber');
```
Comment (jansule) [inline src/Component/Symbolizer/Field/LineDashField/LineDashField.spec.tsx:53]: please remove if not needed

File: src/Component/Symbolizer/Field/LineDashField/LineDashField.spec.tsx [hunk 6]
```diff
@@ -50,9 +50,9 @@ describe('OffsetField', () => {
...
     expect(buttons.length).toBe(2);
-    expect(numberInputs.length).toBe(dashArray.length);
+    // expect(numberInputs.length).toBe(dashArray.length);
```
Comment (jansule) [inline src/Component/Symbolizer/Field/LineDashField/LineDashField.spec.tsx:55]: please remove if not needed

**Review by dnlkoch — APPROVED** [review]

**Review by jansule — APPROVED** [review]

**Pull request merged**